extensions/eclipse-che-theia-dashboard/src/browser/theia-dashboard-client.ts

```diff
@@ -69,9 +70,10 @@ export class TheiaDashboardClient implements FrontendApplicationContribution {
             arrowEl.className = `fa fa-chevron-${this.isExpanded ? 'right' : 'left'}`;
             arrowEl.title = `${this.isExpanded ? 'Show' : 'Hide'} navigation bar`;
         });
+        dashboardEl.appendChild(arrowEl);
```

Comment:
Needs to be fixed depending on logic or return this line back

Reply:
thanks, fixed